src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -327,14 +327,14 @@ protected virtual JToken CancelTransaction(JArray _params)
             TransactionState state = NativeContract.Ledger.GetTransactionState(system.StoreView, txid);
             if (state != null)
             {
-                throw new RpcException(32700, "This tx is already confirmed, can't be cancelled.");
+                throw new RpcException(RpcErrorFactory.BadRequest("This tx is already confirmed, can't be cancelled."));
```

Comment:
Now I see what's the problem with it. `BadRequest` is not exactly the code that should be used here, because it's returned on request parsing errors. @Liaojinghui, how about `AlreadyExists` error (-501) with proper error's `data` set?